### ISSUE TITLE
ClassNotFoundException on Gradle 8.11

### ISSUE THREAD

**Issue by kyay10** [xvik/gradle-animalsniffer-plugin#100]
Getting the error:
```
Could not determine the dependencies of task ':arrow-annotations:check'.
> Could not create task ':arrow-annotations:animalsnifferMain'.
   > Could not create task of type 'AnimalSniffer'.
      > Could not generate a decorated class for type AnimalSniffer.
         > org/gradle/api/reporting/internal/TaskReportContainer
```
This only happens on 8.11, and seems to be because the plugin tries to use `TaskReportContainer`

**Comment by xvik** [comment]
Thank you for the info! 
Problem confirmed, but it'll take some time to fix. Plus, most likely, this release will also include android support (contributed yesterday), which will also take time.

**Comment by xvik** [comment]
[released 1.7.1](https://github.com/xvik/gradle-animalsniffer-plugin/releases/tag/1.7.2) hotfix version (mentioned android support would be a part of upcoming 2.0)
